BWTSolver.cs

```diff
@@ -0,0 +1,92 @@
+namespace Burrows_Wheeler_Transform;
+
+/// <summary>
+/// Class containing Burrows-Wheeler transform methods.
+/// </summary>
+public class BWTSolver
+{
+/// <summary>
+/// Rearranges a string with Burrows-Wheeler transform algorithm.
+/// </summary>
+/// <param name="input">A string to transform.</param>
+/// <returns>Transformed string.</returns>
+    public static (string, int) BurrowsWheelerTransform(string? input)
```

Comment:
По стайлгайду надо именовать элементы кортежа: с большой буквы, после названий типов
```suggestion
    public static (string ..., int ...) BurrowsWheelerTransform(string? input)
```

Reply:
У меня стайлкоп ругается и требует CamelCase((